t/t1011-job-archive-interface.t

```diff
@@ -81,6 +81,37 @@ test_expect_success 'check that no jobs show up under user' '
 	grep -f no_jobs.expected no_jobs.test
 '
 
+# This set of four tests ensures that a job that has no 'alloc' event and
+# therefore no 't_run' timestamp is properly handled by the fetch-job-records
+# command.
```

Comment:
Suggestion: Amend the comment to be clear we're testing the case where a job has an _R_ in the KVS but no `alloc` event, and therefore the job did not enter the RUN state and has no `t_run` timestamp. We put a fake R in the KVS after canceling a job to simulate this condition, which may occur in practice when a job is canceled between after the KVS commit of R is started but before it completes and the `alloc` event is posted.
(See https://github.com/flux-framework/flux-core/issues/7172)